docs/en/about.md

```diff
@@ -1,10 +1,13 @@
 ---
-layout: page
+layout: default
 title: About
-permalink: /about/
-advertise: About
+nav_link: About User Sync
```

Comment:
Remove this file completely.  This Nav Link is actually used for docs/en/index.md, so the About file is never shown.